config.py

```diff
@@ -25,7 +25,8 @@ class DevelopmentConfig(Config):
 class TestingConfig(Config):
     """Configurations for Testing, with a separate test database."""
     TESTING = True
-    SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:admin@localhost/test_db'
+    SQLALCHEMY_DATABASE_URI = 'postgresql://test:admin@localhost/test_db'
```

Comment:
It's good practice to place `SQLALCHEMY_DATABASE_URI` in a separate `.env` file and export it to the application, it prevents security issues